documentation/modules/mtv-aio-buffer.adoc

```diff
@@ -0,0 +1,248 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="mtv-aio-buffer_{context}"]
+= AIO buffer counts for customer usage
+
+This document is intended to summarize the performance team's testing on the impact of changes to the AIO buffer count, specifically to assess any effects on datastore read rates and overall migration time when using the {project-first} product.
+
+.Additional information
+
+* link:https://docs.vmware.com/en/VMware-vSphere/7.0/vsphere-vddk-programming-guide/GUID-5D166ED1-7205-4110-8D72-0C51BB63CC3D.html[Best Practices for NBD Transport]
+* link:https://forums.veeam.com/vmware-vsphere-f24/solution-for-poor-nbd-performance-t93084.html[Solution for poor NBD performance]
+
+[id="mtv-aio-performance-results_{context}"]
+== Performance Testing Results
+
+There were several extensive tests, which were conducted across multiple configurations:
+
+* Single VMware ESXi server compared to multiple ESXi servers.
+* Single virtual machine (VM) compared to multiple VMs.
+
+[id="mtv-aio-buffer-key-findings_{context}"]
+=== Key findings
+
+The best migration performance was achieved by migrating using Multiple 10 VMs(10) on a single ESXi host with the following values:
+
+* `VixDiskLib.nfcAio.Session.BufSizeIn64KB=16`
+* `vixDiskLib.nfcAio.Session.BufCount=4`
+
+The following improvements were noted when using AIO buffer (Asynchronous Buffer Counts) settings:
```

Comment:
I suggest  indent right as it relates to the above Many VMs on single host

Reply:
@dgur - changes made and preview manually updated (Netlify has ended the license). thanks

Reply:
looks good now.
I need now that Tzahi and Mordechai validate the commands to set and remove it